src/components/macros/inventory.tsx

```diff
@@ -0,0 +1,332 @@
+                                                                ii.name === i.name ? { ...ii, name: e.target.value } : ii
+                                                            )
+                                                        };
+                                                    }
+                                                    return s;
+                                                })
+                                                setLocalInventorySelections(newInventorySelections);
+                                                setTimeout(() => {
+                                                    const input = document.getElementById(deckRoomSelections.roomName + "-" + index) as HTMLInputElement;
+                                                    if (input) {
+                                                        input.focus();
+                                                    }
+                                                }, 10);
+                                            }}
+                                        />
+                                    </div>
+                                    <div style={{ display: 'flex', flexDirection: 'column', gap: '2px', flex: 1 }}>
+                                        <label>{"Count"}</label>
+                                        <input
+                                            type="number"
+                                            value={i.count}
+                                            onChange={(e) => {
+                                                const newInventorySelections = localInventorySelections.map((s) => {
+                                                    if (s.roomName === deckRoomSelections.roomName) {
+                                                        return {
+                                                            ...s,
+                                                            inventoryAdded: s.inventoryAdded?.map((ii) =>
+                                                                ii.name === i.name ? { ...ii, count: parseInt(e.target.value) } : ii
+                                                            )
+                                                        };
+                                                    }
+                                                    return s;
+                                                })
+                                                setLocalInventorySelections(newInventorySelections);
+                                            }}
+                                        />
+                                    </div>
+                                    <div style={{ display: "flex", flexDirection: "column", gap: "2px", width: '40px', height: "100%", alignSelf: "center" }}>
+                                        <button style={{ width: '40px', height: '40px' }} onClick={() => {
```

Comment:
Why are you using unstyled buttons here instead of the standard UI buttons that are used everywhere else?